.env

```
@@ -0,0 +1,74 @@
+SORMAS_POSTGRES_PASSWORD=password
+
+SORMAS_POSTGRES_USER=sormas_user
+SORMAS_DOCKER_VERSION=2.14.0
+SORMAS_VERSION=1.52.1
+SORMAS_SERVER_URL=sormas-docker-test.com
+SORMAS_URL=https://github.com/hzi-braunschweig/SORMAS-Project/releases/download/
+
+KEYCLOAK_DB_HOST=keycloak-postgres
+KEYCLOAK_DB_NAME=keycloak
+KEYCLOAK_DB_USER=keycloak
+KEYCLOAK_DB_PASSWORD=password
+KEYCLOAK_DB_VENDOR=postgres
+KEYCLOAK_ADMIN_USER=admin
+KEYCLOAK_ADMIN_PASSWORD=password
+
+KEYCLOAK_SORMAS_UI_SECRET=changeit
+KEYCLOAK_SORMAS_REST_SECRET=changeit
+KEYCLOAK_SORMAS_BACKEND_SECRET=changeit
+
+DB_HOST=postgres
+DOMAIN_NAME=sormas
+DB_NAME=sormas
+DB_NAME_AUDIT=sormas_audit
+DB_JDBC_MAXPOOLSIZE=128
+LOCALE=de-DE
+EPIDPREFIX=DEU
+MAIL_HOST=localhost
+SEPARATOR=\;
+EMAIL_SENDER_ADDRESS=noreply@sormas-docker-test.com
+EMAIL_SENDER_NAME=SORMAS
+LATITUDE=52.2109803
+LONGITUDE=10.5314925
+MAP_ZOOM=10
+MAIL_FROM="root@localhost"
+SORMAS_PATH=/srv/dockerdata/sormas
```

```suggestion
SORMAS_PATH=./docker-data
```
This is how it is done in the other SORMAS projects, so everything stays local to the repo and does not touch your system. Just make sure it's in the .gitignore as well :)

---

Checking this I am getting the following issues for starting the `postgres` container:
`

The files belonging to this database system will be owned by user "postgres".

This user must also own the server process.


The database cluster will be initialized with locale "en_US.utf8".

The default database encoding has accordingly been set to "UTF8".

The default text search configuration will be set to "english".


Data page checksums are disabled.


fixing permissions on existing directory /var/lib/postgresql/data ... ok

creating subdirectories ... ok

selecting default max_connections ... 100

selecting default shared_buffers ... 128MB

selecting default timezone ... Europe/Berlin

selecting dynamic shared memory implementation ... posix

creating configuration files ... ok

running bootstrap script ... ok

sh: locale: not found

2021-01-15 11:21:59.543 CET [30] WARNING: no usable system locales were found

performing post-bootstrap initialization ... ok


WARNING: enabling "trust" authentication for local connections

You can change this by editing pg_hba.conf or using the option -A, or

--auth-local and --auth-host, the next time you run initdb.

syncing data to disk ... ok


Success. You can now start the database server using:


pg_ctl -D /var/lib/postgresql/data -l logfile start


waiting for server to start....2021-01-15 10:22:00.309 GMT [35] LOG: listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"

2021-01-15 10:22:00.356 GMT [36] LOG: database system was shut down at 2021-01-15 10:21:59 GMT

2021-01-15 10:22:00.374 GMT [35] LOG: database system is ready to accept connections

done

server started


/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/setup_sormas.sh

Starting database setup...

CREATE ROLE

CREATE DATABASE

CREATE DATABASE

You are now connected to database "sormas" as user "postgres".

CREATE LANGUAGE

ALTER LANGUAGE

CREATE EXTENSION

CREATE EXTENSION

CREATE EXTENSION

CREATE EXTENSION

GRANT

You are now connected to database "sormas_audit" as user "postgres".

CREATE EXTENSION

NOTICE: extension "plpgsql" already exists, skipping

COMMENT

GRANT

NOTICE: relation "schema_version" does not exist, skipping

ALTER TABLE


waiting for server to shut down....2021-01-15 10:22:01.097 GMT [35] LOG: received fast shutdown request

2021-01-15 10:22:01.109 GMT [35] LOG: aborting any active transactions

2021-01-15 10:22:01.109 GMT [35] LOG: worker process: logical replication launcher (PID 42) exited with exit code 1

2021-01-15 10:22:01.110 GMT [37] LOG: shutting down

2021-01-15 10:22:01.289 GMT [35] LOG: database system is shut down

done

server stopped


PostgreSQL init process complete; ready for start up.


Traceback (most recent call last):

File "/usr/local/bin/alter_system.py", line 143, in <module>

sys.exit(main(sys.argv))

File "/usr/local/bin/alter_system.py", line 139, in main

values = get_tuning_values(config, options.tuning_config)

File "/usr/local/bin/alter_system.py", line 101, in get_tuning_values

cpu = get_cpu()

File "/usr/local/bin/alter_system.py", line 77, in get_cpu

cpu_quota = int(get_cgroup_resources("/sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us"))

File "/usr/local/bin/alter_system.py", line 62, in get_cgroup_resources

with open(file, 'r') as proc_fil:

IOError: [Errno 2] No such file or directory: '/sys/fs/cgroup/cpu,cpuacct/cpu.cfs_quota_us'
`

## Machine used : 
OS: Windows 10 PRO, version 1909, OS Build 18363.1216
Docker:  Engine 20.10.2, Compose 1.27.4, using WSL2 backend